cmd/ejson2env/main.go

```diff
@@ -54,7 +58,7 @@ func main() {
 			fail(fmt.Errorf("no secrets.ejson filename passed"))
 		}
 
-		if err := exportSecrets(filename, keydir, userSuppliedPrivateKey); nil != err {
+		if err := exportSecrets(filename, keydir, userSuppliedPrivateKey, quiet); nil != err {
```

Comment:
I feel like it might make more sense to pass a function to `exportSecrets`, rather than another flag. That would greatly simplify the `exportSecrets` method as well.

For example, if we add the type:

```golang
type ExportFunction func(io.Reader, map[string]string)
```

Then `ExportEnv` and `ExportQuiet` will immediately implement that type. We can then use that type as an argument in `exportSecrets`:

```golang
exportFunc = ExportEnv

if quiet {
   exportFunc = ExportQuiet
}

if err := exportSecrets(filename, keydir, userSuppliedPrivateKey, exportFunc); nil != err {
   fail(err)
}
```

What do you think?

Reply:
Sorry, I meant to write `io.Writer` not `io.Reader` earlier 😅